test-ts/cloud_event_test.ts

```diff
@@ -119,6 +119,16 @@ describe("A 1.0 CloudEvent", () => {
     expect(Object.keys(ce.extensions).length).to.equal(0);
   });
 
+  it("can be constructed with extensions", () => {
+    const extensions = {
+      "extension-key": "extension-value"
+    };
+    const ce = new CloudEvent({
+      extensions, ...fixture
+    });
+    expect(Object.keys(ce.extensions).length).to.equal(1);
```

Comment:
You may want to also check the value in a second `expect()`.

Reply:
yea,  let me add that

Reply:
Added another test in the latest commit